src/components/MailingListForm.js

```diff
@@ -0,0 +1,73 @@
+import React from 'react'
+import addToMailchimp from 'gatsby-plugin-mailchimp'
+
+class MailingListForm extends React.Component {
+  state = {
+    email: '',
+    firstName: '',
+    lastName: '',
+    mailChimpResult: null
+  }
+
+  _handleSubmit = async event => {
+    event.preventDefault()
+
+    // TODO: Validate inputs
+    // TODO: Make only email required
```

Comment:
so does mailchimp require firstname, last name or are you requiring it?

Reply:
Oh Just saw this. Neither require first or last name.